ci/jenkins/test.sh

```diff
@@ -1080,7 +1080,8 @@ fi
 trap clean_antrea EXIT
 if [[ ${TESTCASE} =~ "windows" ]]; then
     if [[ ${TESTCASE} =~ "containerd" ]]; then
-        WINDOWS_YAML_SUFFIX="windows-containerd"
+        # WINDOWS_YAML_SUFFIX="windows-containerd"
```

Comment:
ditto

Reply:
this is a temporary change to run windows CI tests. Will revert it back after review and before merge.